frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -16,25 +16,35 @@ export interface FieldProps {
   validated?: ValidatedOptions;
 }
 
-export interface InputFieldProps extends FieldProps {
-  type: TextInputTypes;
+export interface BaseInputFieldProps extends FieldProps {
   placeholder?: string;
   onChange?: (event) => void;
   onBlur?: (event) => void;
 }
 
+export interface GroupInputProps {
+  beforeInput?: React.ReactNode;
+  afterInput?: React.ReactNode;
+  textType?: TextType;
```

Comment:
Better name it `groupInputType` since we already have a `TextInputType`. Makes it less confusing.